### ISSUE TITLE
Missing the necessary commas between expressions

### ISSUE THREAD

**Issue by hstarorg** [ThisIsManta/vscode-stylus-supremacy#9]
The stylus code: 
```
.project-item {
  border: none;
  box-shadow: 0 1px 3px 0 #d4d4d5, 0 0 0 1px #d4d4d5;
  border-radius: 0.28571429rem;
  transition: box-shadow 0.1s ease, transform 0.1s ease, -webkit-transform 0.1s ease;
}
```
After format will be: 
```
.project-item {
  border: none;
  box-shadow: 0 1px 3px 0 #d4d4d5 0 0 0 1px #d4d4d5;
  border-radius: 0.28571429rem;
  transition: box-shadow 0.1s ease transform 0.1s ease -webkit-transform 0.1s ease;
}
```
It's error styles

**Comment by ThisIsManta** [comment]
Thank you for reporting this. I will take a look ASAP.

**Comment by hstarorg** [comment]
Thank you for your work, it's useful.

**Comment by ThisIsManta** [comment]
Hi @hstarorg,

This has been fixed at version 0.3.3. Please restart your **VS Code** and update the extension.
To prevent this further issues, I have added [the test case](https://github.com/ThisIsManta/stylus-supremacy/commit/9fcdb485faa9257ca0d3c8802ef2dd3c4bfbedff).

Cheers!

**Comment by hstarorg** [comment]
@ThisIsManta Very fast, thank you.
